compiler/natives/src/go/token/position.go

```diff
@@ -1,22 +0,0 @@
-//go:build js
-// +build js
-
-package token
-
-import "sync"
-
-type FileSet struct {
-	mutex sync.RWMutex
-	base  int
-	files []*File
-
-	// replaced atomic.Pointer[File] for go1.19 without generics.
-	last atomicFilePointer
```

Comment:
Removing this override may be premature, since in their current state generics don't support instantiations across package boundaries.

Reply:
It didn't seem to, but I'm still working through the errors for go1.20 in my [WIP PR](https://github.com/gopherjs/gopherjs/pull/1283). There seems to be a few that may be related to removing the crypto overrides and at least one that looks like it may be related to atomic.Pointer.

I was just so excited that generics worked for so many issues that I probably am just being preemptive on this. Like a said before, we can hold off on this. We'll still target a go1.20 with generics. I don't want to add problems while generics is being worked on.